types/parameters/merge_test.go

```diff
@@ -42,12 +43,15 @@ func TestMerge(t *testing.T) {
 	assert.NilError(t, err)
 	parameters, err := Merge(m1, m2, m3)
 	assert.NilError(t, err)
+	fmt.Println(parameters)
 	assert.Check(t, is.DeepEqual(parameters.Flatten(), map[string]string{
 		"foo":      "bar",
 		"bar.baz":  "boz",
 		"bar.port": "10",
 		"bar.foo":  "toto",
-		"baz.0":    "c",
+		"baz.0":    "a",
+		"baz.1":    "b",
+		"baz.2":    "c",
```

Comment:
This is actually a change of behavior. Is this something we want, or should keep the previous "override" behavior ? (there is an option in mergo for that I think)

Reply:
No you are right @simonferquel, if we have parameters overriding a slice, we don't want to append those to the default parameters, but replace it.
@jcsirot can you revert the test modifications?

Reply:
https://github.com/imdario/mergo/pull/112 has been merged into mergo master. I reverted to test and updated the `Merge` function to keep the same behavior.